demoapp/testimony_summary.py

```diff
@@ -0,0 +1,204 @@
+
+'''
+Please pass in open ai api key and bill number as inputs. As of now this program can only work with 3 bills (H3069, H3121, H711)
+
+python testimony_summary.py <<open ai secret key>> <<bill number>>
+
+
+Example command to run this program
+
+python testimony_summary.py sk-my_key H711
+
+'''
+
+import textwrap
+import pandas as pd
+import tiktoken
+import os
+import json
+import sys
+
+from langchain.text_splitter import CharacterTextSplitter
+from langchain.prompts import PromptTemplate
+from langchain.chains import LLMChain
+from langchain.docstore.document import Document
+from langchain.vectorstores import Chroma
+from langchain.embeddings.openai import OpenAIEmbeddings
+from langchain.chains.summarize import load_summarize_chain
+from langchain.schema.runnable import RunnablePassthrough
+from langchain.schema.output_parser import StrOutputParser
+
+from langchain_community.chat_models import ChatOpenAI
+from ipdb import set_trace as st
+from tagging import *
+
+try: 
+    with open('generated_summary.json', 'r') as f: SUMMARY_CACHE = json.load(f)
+except Exception as e: SUMMARY_CACHE = {}
+
+open_ai_key, bill_id = sys.argv[1:]
+os.environ['OPENAI_API_KEY'] = open_ai_key 
+MODEL_NAME = 'gpt-4-1106-preview'
```

Comment:
Define all global constants in a colletion at top of script

Reply:
Do you mean hardcoding the values instead of reading them as arguments?

Reply:
These are already hardcoded, I Just mean to gather the hardcoded constants at the top of the script for easy access, per https://peps.python.org/pep-0008/#constants